src/Chocolatey.Community.Validation.Tests/Rules/DependenciesElementRulesTests.cs

```diff
@@ -13,6 +13,8 @@ public class DependenciesElementRulesTests : RuleTestBase<DependenciesElementRul
         [TestCase("chocolatey", "[,2.0.0)")]
         [TestCase("chocolatey", "[2.0.0, 3.0.0)")]
         [TestCase("chocolatey", "[2.0.0,]")]
+        [TestCase("test-package.hook", null)]
+        [TestCase("test-package.hook", "1.0.0")]
         public async Task ShouldFlagDependencyOnChocolatey(string id, string version)
```

Comment:
Should the name of this test now change?

Reply:
Hmm, now that you have made me look at the name of the method. Then yeah, I guess it should be changed.